config/rbac/role.yaml

```diff
@@ -35,6 +35,14 @@ rules:
   - delete
   - get
   - update
+- apiGroups:
+  - ""
+  resources:
+  - secrets
```

Comment:
Since we are binding this clusterrole as a clusterrolebinding, that means the OLS service account gets all these permissions in all namespaces.

can we constrain this permission to just the `resourceName` we care about?  Or will that break the list/watch permission since list/watch isn't specific to a particular resource name?

Reply:
Yes, we can constrain the resource name to `pull-secret`. A new version is pushed with this modification.